tests/test_bytes.rs

```diff
@@ -0,0 +1,40 @@
+use pyo3::prelude::*;
+use pyo3::types::PyBytes;
+use pyo3::wrap_pyfunction;
+
+mod common;
+
+#[pyfunction]
+fn get_pybytes() -> PyObject {
```

Comment:
Since we recommend users to return `&Py~` types(in this case, `&PyBytes`), I don't think we need this test.

Reply:
Done

Reply:
Actually, it is not clear to me how to return `&PyBytes` directly, see https://gitter.im/PyO3/Lobby?at=5d83e8d456677e767a225079

Reply:
```rust
#[pyclass]
struct ByteSequence {
    elements: Vec<u8>,
}

#[pymethods]
impl ByteSequence {
    fn bytes<'py>(&'py self, py: Python<'py>) -> &'py PyBytes {
        PyBytes::new(py, &self.elements)
    }
}
```

Reply:
I see, thanks. Should this be added as a test?

Reply:
Ok, I saw it already is.